packages/get-platform/src/getPlatform.ts

```diff
@@ -18,16 +17,33 @@ const supportedLibSSLVersions = ['1.0.x', '1.1.x', '3.0.x'] as const
 
 // https://www.geeksforgeeks.org/node-js-process-arch-property/
 export type Arch = 'x32' | 'x64' | 'arm' | 'arm64' | 's390' | 's390x' | 'mipsel' | 'ia32' | 'mips' | 'ppc' | 'ppc64'
+export type DistroInfo = {
+  /**
+   * The original distro is the Linux distro name detected via its release file.
+   * E.g., on Arch Linux, the original distro is `arch`. On Linux Alpine, the original distro is `alpine`.
+   */
+  originalDistro?: string
+
+  /**
+   * The family distro is the Linux distro name that is used to determine Linux flavors based on the same base distro, and likely using the same package manager.
+   * E.g., both Ubuntu and Debian belong to the `debian` family of distros, and thus rely on the same package manager (`apt`).
```

Comment:
This statement isn't true for arch linux I think it uses pacman and not apt by default?